Mathlib/Topology/LocallyConstant/Basic.lean

```diff
@@ -458,7 +458,7 @@ variable [TopologicalSpace Y]
 This definition only makes sense if `f` is continuous,
```

Comment:
This is wrong too now, right? Can you reference the refactor PR in the PR description?